apis/cluster/definition.yaml

```diff
@@ -0,0 +1,157 @@
+                properties:
+                  id:
+                    type: string
+                    description: ID of this Cluster that other objects will use to refer to it.
+                  region:
+                    type: string
+                    description: Region is the region you'd like your resource to be created in.
+                  iam:
+                    type: object
+                    description: IAM configuration to connect as ClusterAdmin.
+                    properties:
+                      roleArn:
+                        description: The IAM Role ARN to connect as ClusterAdmin.
+                        type: string
+                      userArn:
+                        description: The IAM User ARN to connect as ClusterAdmin.
+                        type: string
+                  networkSelector:
+                    type: string
+                    description: NetworkSelector employs a specific type of network architecture.
+                    enum:
+                    - basic
+                    default: basic
+                  serviceSelector:
+                    type: string
+                    description: ServiceSelector employs a specific type of service architecture.
+                    enum:
+                    - default
+                    - gitops-master
+                    default: default
+                  deletionPolicy:
+                    description: When the Composition is deleted, delete the AWS resources. Defaults to Delete
+                    enum:
+                    - Delete
+                    - Orphan
+                    type: string
+                    default: Delete
+                  providerConfigName:
+                    description: Crossplane ProviderConfig to use for provisioning this resources
+                    type: string
+                    default: default
+                  version:
+                    type: string
+                    description: Kubernetes version of the Cluster
+                    enum:
+                    - "1.28"
+                    - "1.27"
+                    - "1.26"
+                    - "1.25"
+                    default: "1.27"
+                  nodes:
+                    type: object
+                    description: Cluster node configuration parameters.
+                    properties:
+                      count:
+                        type: integer
+                        description: Desired node count, from 1 to 100.
+                      instanceType:
+                        type: string
+                        description: instance types associated with the Node Group.
+                        default: t3.small
+                    required:
+                    - count
+                    - instanceType
+                  gitops:
+                    type: object
+                    description: GitOps configure external gitops system for mcp controlplane
```

Comment:
mcp again, it will confuse standard Crossplane/uxp users

Reply:
yes will do it